mobx_codegen/pubspec.yaml

```diff
@@ -14,7 +14,8 @@ environment:
 dependencies:
   path: ^1.6.2
   source_gen: ^0.9.4
-  mobx: ^0.3.3
+  mobx:
+    path: ../mobx
```

Comment:
how about `dependency_overrides`

Reply:
I opened #248. Merge that, publish the alpha, and I'll get this changed to reflect it.

Reply:
published `0.3.6-alpha`

Reply:
you can use the alpha version now (`0.3.6-alpha`)

Reply:
👍 thanks!